index.html

```diff
@@ -0,0 +1,208 @@
+      <a href="index.html">
+        <img src="images/logo-img.png" alt="youth in tech logo" class="web-logo" width="50" height="50">
+      </a>
+      <ul class="navbar-desktop">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#program">Program</a></li>
+        <li><a href="#join-us">Join</a></li>
+        <li><a href="#partners">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li class="tech-campaign">
+          <a href="#">AYT Forum</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <nav class="mobile-menu">
+    <div class="nav-menu">
+      <a href="index.html" class="mobile-menu-logo">
+        <img class="menu-logo" src="images/logo-img.png" alt="youth in tech logo" width="80" height="80"
+        >
+      </a>
+      <img src="icons/Icon-cancel.svg"
+      alt="close button"
+      class="close-button"
+      width="30"
+      height="30">
+    </div>
+    <ul class="menu-ul">
+      <li id="nav-item"><a href="about.html">About</a></li>
+      <li id="nav-item"><a href="#program">Program</a></li>
+      <li id="nav-item"><a href="#join-us">Join</a></li>
+      <li id="nav-item"><a href="#partners">Sponsor</a></li>
+      <li id="nav-item"><a href="#">News</a></li>
+      <li id="nav-item" class="tech-campaign"><a href="#" >AYT Forum</a></li>
+    </ul>
+  </nav>
+  <main>
+    <section class="mobile-container-one">
+      <div class="navbar-mobile">
+        <img src="icons/Icon-Menu.png" alt="hambuger menu">
+      </div>
+      <div class="mobile-one-content">
+        <p class="greetings-mobile">  "Hello! African Youths"</p>
+        <h1>2023 African Youth in Tech Forum</h1>
+        <p class="details">
+          A unique opportunity for the African youths and other young people across the world to learn from the best minds in the world of technology. The event is a tech forum that will be taking place in February, 2023 for 3 days from 3rd Friday to 5th Sunday.
+        </p>
+        <h2>03.01.2023(FRI)~5(SUN)</h2>
+        <p class="venue">@ Kenyatta International Convention Centre, Nairobi, Kenya</p>
+      </div>
+    </section>
+    <section id="program" class="main-program-section"> 
+      <h2>Main Program</h2>
+      <div class="daily-activities">
+        <div class="main-program">
+          <img src="images/open-forum.png" width="40" height="40" alt="door opened">
+          <h3>Forum Opening</h3>
+          <div class="main-paragraph"> 
+            <p>
+              Opening of the forum by the master ceremony accompanied by slow music to invite the visitors.
+            </p>
+          </div>
+        </div>
+        <div class="main-program">
+          <img src="images/exhibition.png" width="40" height="40" alt="exhibition" />
+          <h3>Exhibitions</h3>
+          <div class="main-paragraph">
+            <p>
+              The various technologies built by the youth across the continent are showcased. 
+            </p>
+          </div>
```

Comment:
- [ ] @chepkok3 what the previous reviewer was trying to explain are you ensure all exhibition elements have equal width and height, check the comparison below between your element and the design given, in this case the reviewer pointed out the exhibition element as having a smaller width compared to others:


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/211854418-2aa11bf2-54a3-4ebf-85ee-fb952ed10e1f.png) |  ![image](https://user-images.githubusercontent.com/61408860/211854503-eb2192a8-be03-4f9b-85c1-9c0107264582.png)

`NB this section needed to have only five elements to match the design given`